envoy/api/v2/discovery/eds.proto

```diff
@@ -151,7 +151,7 @@ message EndpointLoadMetricStats {
 
 // These are stats Envoy reports to GLB every so often. Report frequency is
 // defined by
-// :ref:`LoadStatsResponse.load_reporting_interval<envoy_api_field_LoadStatsResponse.load_reporting_interval>`.
+// :ref:`LoadStatsResponse.load_reporting_interval<envoy_api_field_discovery.LoadStatsResponse.load_reporting_interval>`.
 // Stats per upstream region/zone and optionally per subzone.
 // [#not-implemented-hide:] Not configuration. TBD how to doc proto APIs.
 message UpstreamLocalityStats {
```

Comment:
While you're doing this, can we split out the load stats reporter related protos from EDS? It's kind of independent now.

Reply:
How about moving `LbEndpoint`, load stats, and `ClusterLoadAssignment` to base in a file `endpoint.proto`, or alternatively to a package `endpoint`?

Reply:
Hmm, I'm thinking the load stats report is another call-out service (unlike discovery xDS) that is a peer of RLS/auth etc? We can have a `load_stats` subpackage if needed

Reply:
Yes, that seems reasonable. Perhaps, LoadStatsService in `load_stats` package that is distinct from EndpointsDiscoveryService?

Reply:
Done, created `LoadReportingService` in `load_stats` package.